CookBlog.App/Pages/Posts/PostOverview.cs

```diff
@@ -24,7 +24,7 @@ protected override async Task OnInitializedAsync()
     private async Task OpenDialog()
     {
         var options = new DialogOptions { CloseOnEscapeKey = true };
-        var dialog = DialogService.Show<AddPostDialog>("Post Quick Add", options);
+        var dialog = DialogService.Show<AddPostDialog>("", options);
```

Comment:
co tam jest w 1 parametrze?

Reply:
to Tytul, ktory jest wyswietlany na samej gorze w dodaniu